operator/k8s_cep_gc.go

```diff
@@ -131,7 +131,7 @@ func doCiliumEndpointSyncGC(ctx context.Context, once bool, stopCh chan struct{}
 			}
 			if exists {
 				switch pod := podObj.(type) {
-				case *slim_corev1.Node:
+				case *cilium_v2.CiliumNode:
```

Comment:
Unrelated to this change itself, but maybe extend the error message at line 148 to `Saw %T object while expecting *slim_corev1.Pod or *cilium_v2.CiliumNode`?